### PR TITLE
Deprecated org.janusgraph.graphdb.schema

### PR DESCRIPTION
**Pull request by farodin91**
Part of #1853

-----

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### CODE REVIEW

File: janusgraph-core/src/main/java/org/janusgraph/graphdb/schema/EdgeLabelDefinition.java [hunk 1]
```diff
@@ -20,7 +20,10 @@
 
 /**
  * @author Matthias Broecheler (me@matthiasb.com)
+ *
+ * @deprecated part of the management revamp in JG, see https://github.com/JanusGraph/janusgraph/projects/3.
```
Comment (li-boxuan) [inline janusgraph-core/src/main/java/org/janusgraph/graphdb/schema/EdgeLabelDefinition.java:24]: I am just trying to understand: how is this related to the management revamp? I can see even in v0.5 branch, these classes are only used in tests.

Comment (farodin91) [reply janusgraph-core/src/main/java/org/janusgraph/graphdb/schema/EdgeLabelDefinition.java:24]: You could you use the explore schema. I don't think anyone is using it.

Comment (li-boxuan) [reply janusgraph-core/src/main/java/org/janusgraph/graphdb/schema/EdgeLabelDefinition.java:24]: Can you elaborate on "explore schema"? Never heard of it.

Comment (farodin91) [reply janusgraph-core/src/main/java/org/janusgraph/graphdb/schema/EdgeLabelDefinition.java:24]: You could you use the schema provider to list all vertices, edges or properties.

Comment (li-boxuan) [reply janusgraph-core/src/main/java/org/janusgraph/graphdb/schema/EdgeLabelDefinition.java:24]: Got it.

**Review by li-boxuan — APPROVED** [review]
LGTM 👍

**Pull request merged**